e2e/core/pages/notification.page.ts

```diff
@@ -46,18 +46,6 @@ export class NotificationDemoPage {
         return BrowserActions.getText(this.notificationConfig);
     }
 
-    async isNotificationSnackBarDisplayed(): Promise<boolean> {
```

Comment:
@jatin2008 I see you added some files in last commit which probably should not be here?

Reply:
Yeah, that's why we all should use rebase instead of merge to update branches